packages/datagateway-dataview/src/page/pageContainer.component.test.tsx

```diff
@@ -240,4 +245,26 @@ describe('PageContainer - Tests', () => {
 
     expect(history.location.search).toBe('?view=card');
   });
+
+  it('displays warning label when browsing data anonymously', () => {
+    const response = { username: 'anon/anon' };
+    readSciGatewayToken.mockReturnValueOnce(response);
```

Comment:
Is there a reason why this line doesn't need the `as jest.Mock` type assertion whereas the next test does? Just wondering because it looks a little inconsistent

Reply:
This was when I was trying to fix some of the error highlighting VSCode was giving. I will change this one as well.